src/app/app.module.ts

```diff
@@ -4,20 +4,22 @@ import { AngularFireModule } from '@angular/fire';
 import { AngularFireDatabaseModule } from '@angular/fire/database';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
-import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { SharedModule } from './shared/shared.module';
 import { CoreModule } from './core/core.module';
 import { environment } from '../environments/environment';
 
+import { RouterModule } from '@angular/router';
```

Comment:
please follow the next files import structure:
•	libs/angular
•	external app modules (with absolute path)
•	internal module comonents/services